style.css

```diff
@@ -84,4 +85,101 @@ form input {
         width: 1em;
     }
 
+}
+
+/* side nav bar */
+
+.earth-icon {
+    font-size: inherit;
+    width: 1em;
+    fill: var(--gray-2);
+    position: absolute;
+    left: 0.2em;
+    bottom: 0.5em;
+}
+
+.info-icon {
+    font-size: inherit;
+    fill: var(--gray-2);
+    position: absolute;
+    width: 1em;
+    right: 0.5em;
+    bottom: 2.7em;
+}
+
+.star-icon {
+    font-size: inherit;
+    width: 1em;
+    fill: var(--gray-2);
+    position: absolute;
+    left: 0;
+    bottom: 0.7em;
+}
+
+.side-navbar {
+    height: 100vh;
+    width: 160px;
+    left: 0;
+    background-color: var(--white);
+    font-size: 15px;
+    overflow-x: hidden;
+    padding-top: 17px;
+    padding-left: 4em;
+    border-right: solid 1px var(--gray-4);
+}
+
+nav li {
```

Comment:
This is too generic. It will select all `li` items in all `<nav>` elements. I'm pretty sure you only want the sidebar ones.

Let's add a class name and select by that. We could name it `sidebar-item`.